### PR TITLE
[15.0] [MIG] attribute_set

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@ForgeFlow

### CODE REVIEW

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Review by DavidJForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by JuanyDForgeflow — APPROVED** [review]
LGTM

File: attribute_set/models/attribute_attribute.py [hunk 1]
```diff
@@ -247,13 +246,12 @@ def domain_change(self):
             except ValueError:
                 raise ValidationError(
                     _(
-                        """ "{}" is an unvalid Domain name.\n
+                        """ "%(domain)s" is an unvalid Domain name.\n
```
Comment (rousseldenis) [inline attribute_set/models/attribute_attribute.py]: @JasminSForgeFlow Don't get back to old-style python formatting.

You should let the former notation. But you should call format on the translation result (and not inside translation function)

File: attribute_set/models/attribute_set_owner.py [hunk 2]
```diff
@@ -61,12 +61,12 @@ def _insert_attribute(self, arch):
...
-                    """It is impossible to add Attributes on "{}" xml view as there is
+                    """It is impossible to add Attributes on "%(form_name)s" xml
+                    view as there is
                     not one "<separator name="attributes_placeholder" />" in it.
-                    """.format(
-                        form_name
-                    )
+                    """
                 )
+                % ({"form_name": form_name})
```
Comment (rousseldenis) [inline attribute_set/models/attribute_set_owner.py]: Same as above

File: attribute_set/models/attribute_attribute.py (hunk 1 above)
Comment (JasminSForgeFlow) [reply attribute_set/models/attribute_attribute.py]: Updated, Thanks

File: attribute_set/models/attribute_set_owner.py (hunk 2 above)
Comment (JasminSForgeFlow) [reply attribute_set/models/attribute_set_owner.py]: Updated, Thanks

**Pull request merged**